modules/plugins/dashboard/alpha/config.nix

```
@@ -5,216 +5,43 @@
-      			end
-      		end
-
-      		local shortcut = tostring(i + start - 1)
-
-      		local file_button_el = file_button(fn, shortcut, short_fn)
-      		tbl[i] = file_button_el
-      	end
-      	return {
-      		type = "group",
-      		val = tbl,
-      		opts = {},
-      	}
-      end
-
-      local default_header = {
-      	type = "text",
-      	val = {
-
-      		[[███    ██ ███████  ██████  ██    ██ ██ ███    ███]],
-      		[[████   ██ ██      ██    ██ ██    ██ ██ ████  ████]],
-      		[[██ ██  ██ █████   ██    ██ ██    ██ ██ ██ ████ ██]],
-      		[[██  ██ ██ ██      ██    ██  ██  ██  ██ ██  ██  ██]],
-      		[[██   ████ ███████  ██████    ████   ██ ██      ██]],
-
-      		-- [[                               __                ]],
-      		-- [[  ___     ___    ___   __  __ /\_\    ___ ___    ]],
-      		-- [[ / _ `\  / __`\ / __`\/\ \/\ \\/\ \  / __` __`\  ]],
-      		-- [[/\ \/\ \/\  __//\ \_\ \ \ \_/ |\ \ \/\ \/\ \/\ \ ]],
-      		-- [[\ \_\ \_\ \____\ \____/\ \___/  \ \_\ \_\ \_\ \_\]],
-      		-- [[ \/_/\/_/\/____/\/___/  \/__/    \/_/\/_/\/_/\/_/]],
-      	},
-      	opts = {
-      		position = "center",
-      		hl = "Type",
-      		-- wrap = "overflow";
-      	},
-      }
-
-      local section_mru = {
-      	type = "group",
-      	val = {
-      		{
-      			type = "text",
-      			val = "Recent files",
-      			opts = {
-      				hl = "SpecialComment",
-      				shrink_margin = false,
-      				position = "center",
-      			},
-      		},
-      		{ type = "padding", val = 1 },
-      		{
-      			type = "group",
-      			val = function()
-      				return { mru(0, cdir) }
-      			end,
-      			opts = { shrink_margin = false },
-      		},
-      	},
-      }
+    vim.pluginRC.alpha = let
+      setupOpts =
+        if themeDefined
+        then lib.generators.mkLuaInline "require'alpha.themes.${cfg.theme}'.config"
+        else
+          (with cfg; {
+            inherit layout opts;
+          });
+    in ''
+      require('alpha').setup(${toLuaObject setupOpts})
+    '';
 
-      local buttons = {
-      	type = "group",
-      	val = {
-      		{ type = "text", val = "Quick links", opts = { hl = "SpecialComment", position = "center" } },
-      		{ type = "padding", val = 1 },
-          -- TODO: buttons should be added based on whether or not the relevant plugin is available
-      		dashboard.button("e", "  New file", "<cmd>ene<CR>"), -- available all the time
-      		dashboard.button("SPC F", "󰈞  Find file"), -- telescope
-      		dashboard.button("SPC ff", "󰊄  Live grep"), -- telescope
-      		dashboard.button("SPC p", "  Projects"), -- any project
-      		dashboard.button("q", "󰅚  Quit", "<cmd>qa<CR>"), -- available all the time
-      	},
-      	position = "center",
+    assertions = [
+      {
+        assertion = themeDefined || layoutDefined;
+        message = ''
+          You should either define `theme` or `layout`.
```

```suggestion
          One of 'theme' or 'layout' should be defined in Alpha configuration.
```

Assertion messages don't contain formatting options, so using backticks is unnecessary.